CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ option(TBB_NO_APPCONTAINER "Apply /APPCONTAINER:NO (for testing binaries for Win
 option(TBB4PY_BUILD "Enable tbb4py build" OFF)
 option(TBB_CPF "Enable preview features of the library" OFF)
 option(TBB_FIND_PACKAGE "Enable search for external oneTBB using find_package instead of build from sources" OFF)
+option(TBB_TBBBIND_BUILD "Enable building TBBbind libraries" ON)
```

Comment:
Add this option conditionall? For example,
```cmake
if (APPLE)
    message(STATUS "TBBBind build targets are disabled due to unsupported environment")
    set(TBB_TBBBIND_BUILD FALSE)
else()
    option(TBB_TBBBIND_BUILD "Enable building TBBbind libraries" ON)
endif()
```
This will allow not having any messages below, but have simple thing instead:
```cmake
if (TBB_TBBBIND_BUILD)
    add_subdirectory(src/tbbbind)
endif()
```
What do you think about this approach?

Reply:
I think it is a good idea! Please take a look at the patch and review the new logic.